R/guide-axis-theta.R

```diff
@@ -160,12 +160,16 @@ GuideAxisTheta <- ggproto(
 
   build_labels = function(key, elements, params) {
 
+    if (inherits(elements$text, "element_blank")) {
+      return(zeroGrob())
+    }
+
     key <- vec_slice(key, !vec_detect_missing(key$.label %||% NA))
 
     # Early exit if drawing no labels
     labels <- key$.label
     if (length(labels) < 1) {
-      return(list(zeroGrob()))
```

Comment:
I realised downstream code does *not* expect a list of grobs, just a single one.